lib/Target/Jaxite/JaxiteEmitter.cpp

```diff
@@ -229,6 +233,54 @@ LogicalResult JaxiteEmitter::printOperation(tensor::FromElementsOp op) {
   return success();
 }
 
+// Loading variables.
+// Example: temp_nodes[idx] = input[i]
+LogicalResult JaxiteEmitter::printOperation(memref::LoadOp op) {
```

Comment:
Below, you're allocating memrefs in the IR as a flattened array of `types.LweCiphertext`s - given that, when you perform a `memref::LoadOp`, don't you want to flatten the index expression like `input[1 + 2*dim0]` instead of printing out `input[1][2]`?